src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -59,6 +61,10 @@ public bool HasContainer
 
 		public IServiceProvider? Services => MauiContext?.Services;
 
+		public virtual bool NeedsContainer { get; }
```

Comment:
@jsuarezruiz I am not sure if this is correct yet, but this is a virtual instance property because we need a way to override this - I think. For example, the base view adds clipping, and that works. If a label comes along and needs to also influence this, then that handler can do its own logic and then `||` it with the base.

Base:

```csharp
bool NeedsContainer => clip != null;
```

Label:

```csharp
bool NeedsContainer => Background != null || base.NeedsContainer;
```

Reply:
Looks good for me.